lib/sshkit/command.rb

```diff
@@ -182,7 +182,7 @@ def umask(&_block)
 
     def group(&_block)
       return yield unless options[:group]
-      "sg #{options[:group]} -c \\\"%s\\\"" % %Q{#{yield}}
+      "sg #{options[:group]} -c \"%s\"" % %Q{#{yield}}
```

Comment:
Since we're already looking at this part of the code, I think there are some additional changes we can do to make this much easier to read.

First, I want to point out that:

* `%Q{#{yield}}` is the same as `"#{yield}"`
* `"#{yield}"` is the same as `yield.to_s`
* We always pass a string in the block, so `yield.to_s` is the same as `yield`

Taking this further:

* `"... %s ..." % yield` is the same as `"... #{yield} ..."`
* `"... \" ..."` can be written without escapes as e.g. `%Q(... " ...)`

And so this whole line could be much simplified to:

```ruby
%Q(sg #{options[:group]} -c "#{yield}")
```